maddy.conf

```diff
@@ -0,0 +1,44 @@
+# Location of TLS certificate and private key. Global directive is used for all
+# endpoints.
+tls cert_file_path pkey_file
+# or if you are just testing maddy - you can run it without TLS.
+# however you also need to remove smtps:// and imaps:// listening addresses from
+# endpoints below.
+#tls off
```

Comment:
I'd prefer not to have commented out parts. If we want to demonstrate different configurations for other use-cases, I'd rather have separate example files.